modules/ROOT/pages/delete-api-task.adoc

```diff
@@ -1,12 +1,22 @@
-= Removing an API instance on API Manager
+= Remove API Instances
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-The procedure removes the capability to manage an API Version from API Manager. If the API resides in Exchange or on the file system as a ZIP, it isn't deleted from Exchange or the file system.
+You can remove an API instance or a version of an API instance if you no longer need to manage it from API Manager. When you remove the API version from API Manager, the API is not deleted from Anypoint Exchange (Exchange) or from the file system, if it exists in any of these locations. 
 
-. In Anypoint Platform, click API Manager.
-. In API Administration, click the API version link or an instance link.
-. Select Delete from the Actions dropdown.
+Furthermore, the API asset in Exchange that the API references is not deleted.
+
+[Note] 
+--
+You cannot delete an API specification that is currently used in one or more API instances or versions being managed in API Manager. 
+--
+
+To remove API instances or API versions from API Manager:
+
+. Go to *Anypoint Platform* > *API Manager*.
+. In *API Administration*, click the *API version* link or an instance link.
+. Select *Delete* from the *Actions* dropdown.
+. Click *Delete* again to confirm the delete action. 
 +
-. Respond to the prompt to confirm the deletion. Click Delete.
\ No newline at end of file
+Your API instance is now successfully removed.
```

Comment:
GLOBALs:

Choose between "delete" and "remove." Because the UI uses the former, I suggest replacing all "remove"s with "delete"s, unless you clarify a difference.

Clarify what is being deleted: an API instance? a version instance? a version? The wording above is confusing because it uses these alternatively and seemingly interchangeably. Provide a REASON for doing this. Do not mention deleting a specification if it is separate from instance/version; that is off topic. Ditto "asset": why would an asset be removed?